packages/web/src/common/store/social/tracks/sagas.ts

```diff
@@ -587,6 +589,7 @@ export function* watchRecordListen() {
       if (isNativeMobile) return
       console.debug('Listen recorded for track', action.trackId)
 
+      yield* call(waitForBackendSetup)
```

Comment:
So couldn't we get rid of the `waitForAccount`s since the account status is guaranteed to exist if the backend setup is complete?

Reply:
Yes, though I have some concerns over having the same assumption come back to bite us in the future. I'm thinking of changing it to one call that does both like `waitForBackendAndAccount` just to be explicit